src/Mod/Part/App/PartFeature.h

```diff
@@ -52,6 +53,18 @@ class PartExport Feature : public App::GeoFeature
 {
     PROPERTY_HEADER_WITH_OVERRIDE(Part::Feature);
 
+public:
+    enum class GetShapeOption
+    {
+        NoFlag = 0,
+        NeedSubElement = 1,
+        ResolveLink = 2,
+        Transform = 4,
+        NoElementMap = 8,
+        DontSimplifyCompound = 16
+    };
+    using GetShapeOptions = Base::Flags<GetShapeOption>;

```

Comment:
Maybe remove `Get` so just `ShapeOption`.

`Part::Feature::GetShapeOption::NeedSubElement` is super long. I'd do as @kadet1090 suggested and add `using enum class`, making it `Part::Feature::NeedSubElement, or move it out of class so it would be `Part::GetShapeOption::NeedSubElement`.